editoast/src/views/layers/mvt_utils.rs

```diff
@@ -19,33 +21,45 @@ pub struct GeoJsonAndData {
     pub data: JsonValue,
 }
 
+fn geometry_mvt_geom_type(geometry: &Geometry) -> GeomType {
```

Comment:
suggestion: I'd be nice to have some sort of idiomatic naming (`from`, or `into` in the name of the function). I see it's impossible to either implement `Into` or `From` in this case, but we can still have the function naming be explicit about that. Something like `geometry_into_geom_type`?

Or if you want to go a step further (but it might be overkill):

```rust
trait FromGeometry {
  fn from_geometry(geometry: &Geometry) -> Self;
}
impl FromGeometry for GeomType {
  // ...
}
```

Reply:
I went with the name change as it's just a local function with one usage.